fib-service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: fibonacci-app
```

Comment:
did you mean to change the name?

Reply:
yes, in the case that they don't delete it, i don't want these to conflict.

Reply:
oh, I thought we'd just upgrade the same app to include the builder.  Then they're slowly growing the app's infrastructure as they grow their knative expertise. Similar to how we keep the same app when we switch to knctl.

Reply:
okay - updated.